postgres/seed.sql

```diff
@@ -1,20 +1,20 @@
 -- ========================================
--- Seed users
+-- Seed users with username and hashed password
+-- Passwords are hashed; original password in comment

```

Comment:
Update lại chỗ này, không còn mật khẩu và username.

Ngoài ra em có thể update câu lệnh sql như thế này

```sql
INSERT INTO users ("firstName", "lastName", "email")
VALUES
    ('Alice', 'Nguyen', 'alice@example.com'), 
    ('Bob', 'Tran', 'bob@example.com'), 
    ('Charlie', 'Le', 'charlie@example.com'), 
    ('David', 'Pham', 'david@example.com'), 
```

Mấy field còn lại thì postgres nó tự động điền dữ liệu rồi. 

Ở đây "createdAt" và "updatedAt" là hoàn toàn OK vì mặc định thì postgress cũng dùng current timestamp để điền giá trị

Riêng id thì làm thế này cũng được nhưng nó hạn chế là không chủ động được giá trị id, ví dụ mình hardcode user -  id ở nơi khác thì không được. Nói chung là tuỳ